### PR TITLE
fix(curriculum): Update data analysis slides and question for first section

### LINKED ISSUE #42343: Data Analysis with Python Misleading Quiz Question
**Affected page**
https://www.freecodecamp.org/learn/data-analysis-with-python/data-analysis-with-python-course/introduction-to-data-analysis

**Issue**
This question implies that R is better for data science than Python. This is a pretty debatable point, and not really a question to gauge the learner's level of understanding.

> Why should you choose R over Python for data analysis?

We can mitigate this issue somewhat by changing the text of the question to something more neutral, such as

> Why might you choose to use R over Python for data analysis?

Though, really this seems like a poor question to summarize the contents of the slide deck/presentation.

I think we should either update the question to more neutral language or (preferably) pick a better question to summarize the important portions of this lesson (language wars are not 'important' content for us to quiz users on, IMHO).

---

Speaking of the slides,

They say:

From the slides for this lesson:
> Python, sadly, is not always the answer

> * When R Studio is needed
> * When dealing with advanced statistical methods
> * When extreme performance is needed

It is a dubious claim that R is more performant than Python. The performance difference between the two depends upon the problem at hand, the skill of the programmer, the programming paradigmns used, etc. For some situations, Python can be much faster than R.

With the explosion of AI/ML/Data Science packages in Python in recent years, it is also somewhat dubious that R is better for 'advanced statistical methods', as the range of analysis packages for Python has grown a considerable amount.

Each language has problems they are more commonly used for, but I wouldn't really say one is 'better' in the way this content is implying. The situation is much more grey than the black and white picture the slides paint.

I think this sort of stale content is a risk with our video based lessons, where it is harder to update content as technology changes.

### PR DESCRIPTION
**Pull request by beaucarnes**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #42343

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]

**Comment by raisedadead** [comment]
Hey @jeremylt not sure why I could not request a review from you (maybe it's a GitHub glitch), but it would be cool if you took a look at this too.

**Comment by jeremylt** [comment]
I don't think I have reviewer permissions in this repo.

But, LGTM. I just have one question. When I write exam questions that have the word **not**, I personally like to bold the word to help reduce confusion. Do we want to do that here? Will that interfere with the translation efforts in any way?

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/introduction-to-data-analysis.md [hunk 1]
```diff
@@ -10,29 +10,29 @@ dashedName: introduction-to-data-analysis
...
 
-\- [Slides](https://docs.google.com/presentation/d/1fDpjlyMiOMJyuc7_jMekcYLPP2XlSl1eWw9F7yE7byk)
+\- [Slides](https://docs.google.com/presentation/d/1cUIt8b2ySz-85_ykfeuuWsurccwTAuFPn782pZBzFsU/edit?usp=sharing)
 
 # --question--
 
 ## --text--
 
-Why should you choose R over Python for data analysis?
+Which of the following is not part of Data Analysis?
```
Comment (jeremylt) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/introduction-to-data-analysis.md:19]: Do we want to bold **not**?

**Comment by raisedadead** [comment]
> I don't think I have reviewer permissions in this repo.

Yeah, I just realised that after I made my comment. Sent you an invite. Should be in your inbox. Thanks again.

**Review by jeremylt — DISMISSED** [review]
LGTM for the content in this PR

The attached slides though seem to have that emoji typo: 
> 👎 Extremely Powerful 💪

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/introduction-to-data-analysis.md (hunk 1 above)
Comment (raisedadead) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/introduction-to-data-analysis.md:19]:
```suggestion
Which of the following is **not** part of Data Analysis?
```

Comment (raisedadead) [reply curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/introduction-to-data-analysis.md]: I have applied a commit to make it bold.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
Hey, @beaucarnes other than the typo this is good to go. We will land this assuming you have the ability to edit the slide deck after the fact and when you get a chance.

**Comment by beaucarnes** [comment]
I fixed the typo with the emoji.

**Review by jeremylt — APPROVED** [review]

**Pull request merged**